js/src/cli.ts

```diff
@@ -696,16 +696,80 @@ async function collectFiles(
 // our packages (braintrust and autoevals) external, in case they're
 // installed in a relative path.
 const markOurPackagesExternalPlugin = {
-  name: "make-all-packages-external",
+  name: "make-our-packages-external",
   setup(build: esbuild.PluginBuild) {
-    const filter = /^(braintrust|autoevals|@braintrust\/)/;
-    build.onResolve({ filter }, (args) => ({
+    // Mark our packages as external
+    const ourPackagesFilter = /^(braintrust|autoevals|@braintrust\/)/;
+    build.onResolve({ filter: ourPackagesFilter }, (args) => ({
       path: args.path,
       external: true,
     }));
   },
 };
 
+// Inspired by and modified from https://github.com/evanw/esbuild/issues/1051
+const nativeNodeModulesPlugin = {
+  name: "native-node-modules",
+  setup(build: esbuild.PluginBuild) {
+    // Keep track of packages that contain .node files
+    const nativePackages = new Set<string>();
+
+    // Helper to add a package and its platform-specific variants
+    const addNativePackage = (pkgName: string) => {
+      nativePackages.add(pkgName);
+      if (pkgName.includes("@")) {
+        const [scope, name] = pkgName.split("/");
+        const platformPkgs = [
+          `${scope}/${name}-darwin-arm64`,
```

Comment:
are these standard suffixes?  quick grep looks like it. looks like (by searching our repo as an example and finding in our lock files) there's some variants like -ia32- missing. what determines the targets?

Reply:
I just had an AI generate them. Happy to add to this list if you have specific recommendations or we can keep adding to it as we go

Reply:
just notice this notification 😅 <br><br>no strong feelings here. we can always come back to add more as needed 👍